microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -44,7 +44,9 @@ To use the Microsoft Edge Add-ons API, you need to enable the API for your proje
 
 1. In the **Publish APIs** page, select the **Create API credentials** button to generate the API credentials.  This step may take a few minutes.  After the APIs are enabled, the **Client ID**, **Client Secret**, and **Auth Token URL** are displayed on this page.
 
-1. Note the **ClientID**, **Client Secret** and the **Auth Token URL**.  You'll use them in the next step to get an access token.
+1. Note the **ClientID**, **Client Secret** and the **Auth Token URL**. You'll use them in the next step to get an access token. Note that client secret is visible only once, after enabling or renewing the API. 
+
+![image](https://user-images.githubusercontent.com/11265822/150777593-42cb9d37-0c1f-4ef4-9de6-9c4bc1166b96.png)
 
```

Comment:
Re-capture the image.  Make the window smaller, like 1100 px wide, 700 px tall or preferably smaller.
In another tab or window, go to https://browsersize.com but don't use that webpage's buttons; just drag the window edge until "Browser window size" is right.
Then make the window you want to capture even smaller, as small as possible without breaking the wrapping/ page layout.  Then capture it.
A smaller window before capturing, will produce bigger, more legible text in the rendered article.